packages/pools/src/router/routes.ts

```diff
@@ -188,7 +188,7 @@ export class OptimizedRoutes implements TokenOutGivenInRouter {
         return includedRoutes;
       } else {
         route.pools.forEach(({ id }) => uniquePoolIds.add(id));
-        includedRoutes.unshift(route);
```

Comment:
LGTM to in isolation, though not seeing the corresponding sort line

Reply:
Before they're filtered, they're sorted on line 182

Reply:
Hrmm, thats sorting by path length, not sorting by net price to user though?

Reply:
Yeah it's just sorting by length. Price comparison will happen later

This PR contains a critical bug fix, we can follow up with improvements